### PR TITLE
Update health API readme

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

Updates the health API docs to align with the current avalanchego service.

## How this works

N/A

## How this was tested

N/A

### CODE REVIEW

File: api/health/service.md [hunk 1]
```diff
@@ -90,125 +92,215 @@ This API uses the `json 2.0` RPC format. For more information on making JSON RPC
...
-health.health() -> {
-    checks: []{
-        checkName: {
-            message: JSON,
-            error: JSON,
-            timestamp: string,
-            duration: int,
-            contiguousFailures: int,
-            timeOfFirstFailure: int
-        }
```
Comment (StephenButtolph) [inline api/health/service.md:108]: I felt that this didn't add anything useful. If we wanted to we could include the golang structs defining the inputs and outputs... But I think the example shows it all.

File: api/health/service.md [hunk 2]
```diff
@@ -15,64 +15,66 @@ This API set is for a specific node, it is unavailable on the [public server](/t
...
+curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL&tag=28nrH5T2BMvNrWecFcV3mfccjs6axM1TVyqe79MCv2Mhs8kxiY'
 ```
 
 Returned results will contain checks for both subnetIDs and global health checks.
 
+### Endpoints
+
+The available endpoints for GET requests are:
+
+- `/ext/health` returns a wholeistic report of the status of the node.
```
Comment (abi87) [inline api/health/service.md:72]: did you mean `holistic`?

File: api/health/service.md [hunk 3]
```diff
@@ -15,64 +15,66 @@ This API set is for a specific node, it is unavailable on the [public server](/t
...
+  **Most operators should monitor this status.**
+- `/ext/health/health` is the same as `/ext/health`.
+- `/ext/health/readiness` returns healthy once the node has finished initializing.
+- `/ext/health/liveness` returns healthy once the endpoint is avialable.
```
Comment (abi87) [inline api/health/service.md:76]: did you mean `available`?

**Review by abi87 — APPROVED** [review]
found a couple of possible typos. Other than that LGTM

File: api/health/service.md [hunk 4]
```diff
@@ -15,64 +15,66 @@ This API set is for a specific node, it is unavailable on the [public server](/t
 
 :::
 
+## Health Checks
+
+The node periodically runs all health checks, including health checks for each chain.
+
+The frequency that health checks are run can be specified by [--health-check-frequency](/nodes/configure/avalanchego-config-flags).
```
Comment (meaghanfitzgerald) [inline api/health/service.md:22]:
```suggestion
The frequency that health checks are run can be specified by [--health-check-frequency](/nodes/configure/avalanchego-config-flags.md).
```

File: api/health/service.md [hunk 5]
```diff
@@ -90,125 +92,215 @@ This API uses the `json 2.0` RPC format. For more information on making JSON RPC
...
+    "id": 1
+}
+```
+
+In this example response, the node was able to handle the request and mark the service as healthy.
+
+**Response Explanation:**
+
+- `checks` is an empty list.
+- `healthy` is true.
```
Comment (meaghanfitzgerald) [inline api/health/service.md:306]:
```suggestion
- `healthy` is true.

```

Comment (meaghanfitzgerald) [reply api/health/service.md:306]: markdownlint complains when the doc doesn't end with a blank line

File: api/health/service.md [hunk 6]
```diff
@@ -90,125 +91,215 @@ This API uses the `json 2.0` RPC format. For more information on making JSON RPC
...
 
-### Filtering
+In this example response, every check has passed. So, the node has finished the startup process.
 
-JSON RPC methods in Health API supports filtering by tags. In order to filter results use `tags`
-params in the
-request body. `tags` accepts a list of tags. Currently only `subnetID`s are supported as tags.
-For example, to filter health results by subnetID `29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL`
-use the following request:
+**Response Explanation:**
```
[the remark this answers is not in the document]
Comment (StephenButtolph) [reply api/health/service.md:262]: I talked with @meaghanfitzgerald and she felt that this doc would be more consistent with the rest of the docs if each call were separate... I tend to agree that we should probably try to reduce duplication when possible... but 🤷 I think that this is at least an improvement over the current doc

**Pull request merged**